src/WorkBC.Importers.Federal/Program.cs

```diff
@@ -33,12 +38,23 @@ private static async Task Main(string[] args)
             Parser.Default.ParseArguments<CommandLineOptions>(args)
                 .WithParsed(o => { options = o; })
                 .WithNotParsed(o => { Environment.Exit(0); });
+            
+            string connectionString = configuration.GetConnectionString("DefaultConnection");
+            var dbContext = new JobBoardContext(connectionString);
 
-            var service = new XmlImportService(configuration, options, logger);
+            //get API settings
+            var federalSettings = new FederalSettings();
+            configuration.GetSection("FederalSettings").Bind(federalSettings);
+            var baseUrl = federalSettings.FederalJobXmlRoot;
+            
+
+            var httpClient = CreateHttpClient(configuration);
+            var federalApiService = new FederalApiService(httpClient, baseUrl, logger);
+            var xmlImportService = new XmlImportService(federalApiService, dbContext, options, logger);
```

Comment:
The `XmlmportService` has been refactored.  Method that related to calling the federal job board API have been placed into their own service, `FederalApiService`.  By passing in the `HttpClient` or the `JobBoardContext` it makes the services easier to test.